src/airport.js

```diff
@@ -1,6 +1,48 @@
+const Plane = require('./Plane');
+
 class Airport {
-  // here's a starting point for you
+  landedPlanes = [];
+  landedCapacity;
+
+  // constructor(defaultLandedCapacity) {
+  //   this.landedCapacity = defaultLandedCapacity;
+  // }
 
-}
+  constructor(defaultLandedCapacity) {
+    if (Number.isInteger(defaultLandedCapacity) && defaultLandedCapacity > 0) {
+      this.landedCapacity = defaultLandedCapacity;
+    } else {
+      this.landedCapacity = 0;
+    }
+  }
+  landPlane = plane => {
+    if (plane instanceof Plane && this.getLandedPlaneCount() < this.getLandedCapacity() && !this.isPlanePresent(plane)) {
+      this.landedPlanes.push(plane);
+      // return true;
+      //   } else {
+      //       return false;
+    }
```

Comment:
Had originally returned this a Boolean, however it was unused, and I checked the second landing of a plane (which came in a late test) by using a then existing method getPlaneCount